### ISSUE TITLE
ARCoreInterface does not update the values within a updated plane.

### ISSUE THREAD

**Issue by kweeniemike** [Unity-Technologies/experimental-ARInterface#7]
https://github.com/Unity-Technologies/experimental-ARInterface/blob/8a6b5aa10f7b850714d9cc1849d7b70d24af9b02/Assets/UnityARInterface/Scripts/ARCoreInterface.cs#L289

On a plane update, the ARCoreInterface passes the update with OnPlaneUpdate, but does not update the values of the boundedPlane it passes along to the values of the corresponding TrackedPlane.

**Comment by vtsgarbee** [comment]
same issue here. galaxy S8.
on the other hand the HelloAR scene works flawlessly.

**Comment by kweeniemike** [comment]
That is probably because the HelloAR scene does not use ARInterface. It is part of the GoogleARCore Unity package, and seperate from ARInterface

**Comment by vtsgarbee** [comment]
yeah that's what i was trying to point out :) , the problem lies in the ARInterface layer .
I hope to get a fix or a hack soon, the framework is really easy and useful.

**Comment by kweeniemike** [comment]
I'm currently using the following as a hack/fix. 

At the following line:

https://github.com/Unity-Technologies/experimental-ARInterface/blob/8a6b5aa10f7b850714d9cc1849d7b70d24af9b02/Assets/UnityARInterface/Scripts/ARCoreInterface.cs#L289

Make it so it reads:

```
if (trackedPlane.SubsumedBy == null)
 {
    boundedPlane.center = trackedPlane.Position;
    boundedPlane.rotation = trackedPlane.Rotation;
    boundedPlane.extents = trackedPlane.Bounds;
    OnPlaneUpdated(boundedPlane);
}
```
So basically add those three lines updating boundedPlane.

**Comment by vtsgarbee** [comment]
thanks man, it worked!
it will do for now.
owe you one!

**Comment by jimmy-jam** [comment]
merged pr #18 which includes this
